src/Config.hs

```diff
@@ -19,7 +19,7 @@ data Config = Config
     , _htmlStatic :: FilePath
     , _cfgCsrfSecret :: CsrfSecret
     }
-  deriving (Show) -- , Eq)
+  deriving (Show)
```

Comment:
Thanks. The reason was a missing Eq instance for CsrfSecret which would require another round trip on thentos.